c2cgeoportal/scaffolds/create/mapserver/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM camptocamp/mapserver:latest
```

Comment:
Nope, the ADD /COPY in the Dockerfile can see only the stuff in the directory where the file is located. Very weird limitation IMHO, but a limitation we cannot work around.